src/Components/Option1.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react'
+import './Option.css';
+import { useContext } from 'react';
+import { myContext } from './Home';
+
+export default function Option1() {
+    const [toggle,setToggle] = useContext(myContext);
+    let Border = {
+        border: toggle? '2px solid grey': ''
+      }
+    
+      let borderradius = {
+        borderRadius: toggle? 10: 0
+
+      }
+    return (
+        <div  style={{ ...Border, ...borderradius}}  className='Option' onClick={()=>{
```

Comment:
dont create a new component. Please go through https://reactjs.org/docs/thinking-in-react.html to identify what needs to be a component, what need not.